entries/h/hsbc.com.au.json

```diff
@@ -0,0 +1,19 @@
+{
+  "HSBC [AU]": {
+    "domain": "hsbc.com.au",
+    "img": "hsbc.svg",
+    "tfa": [
+      "custom-software"
+    ],
+    "custom-software": [
+      "HSBC Australia"
+    ],
+    "documentation": "https://www.hsbc.com.au/ways-to-bank/support/digital-secure-key/",
```

Comment:
Are these security codes required to log in or perform certain _sensitive_ actions?

Reply:
Secure code is required to login now